src/chains-config/metadata-consts/index.ts

```diff
@@ -0,0 +1,45 @@
+import { Definition } from '../../types/chains-config';
+import { kusamaDefinitions } from './kusama-consts';
+import { polkadotDefinitions } from './polkadot-consts';
+
+/**
+ * Creates an object that orders each runtime to their appropriate data
```

Comment:
This is too abstract; the code manipulates concrete types (`Definition`) and operates over concrete types (runtimes). I think it's good to state more up front what this does.

Reply:
Agreed, this could use more detailed explanation, and maybe even an output example.